library/Reporting/Web/Forms/ReportForm.php

```diff
@@ -49,7 +50,20 @@ protected function assemble()
             'description' => $this->translate(
                 'A unique name of this report. It is used when exporting to pdf, json or csv format'
                 . ' and also when listing the reports in the cli'
-            )
+            ),
+            'validators' => [
+                'Callback' => function ($value, CallbackValidator $validator) {
+                    if ($value !== null && $value[0] === '.') {
```

Comment:
I think I suggested something like this but it's not enough to protect from path traversal. `/../../owned.pdf` would be valid but that's not something we want. Single dots are no harm even at the beginning as would be just a hidden file. Please research "path traversal" a bit and suggest how to prevent it here.